### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by janpio**
- fix indentation
- add TOC
- remove JIRA link
- integrate changes from https://github.com/apache/cordova-plugin-splashscreen/pull/98/
- add headline for image config instructions
- remove outdated translations
- rework introduction

closes #98

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,70 +6,78 @@ addons:
 
 env:
   global:
-    - SAUCE_USERNAME=snay
```
Comment (timbru31) [inline .travis.yml]: The Travis part should be a separate PR

Comment (janpio) [reply .travis.yml]: yep, fucked up merged. will revert and rebase now.

File: README.md [hunk 2]
```diff
@@ -27,7 +27,26 @@ description: Control the splash screen for your app.
 
 # cordova-plugin-splashscreen
 
-This plugin is required to work with splash screens. This plugin displays and hides a splash screen during application launch.
+This plugin displays and hides a splash screen while your web application is launching. Using its methods you can also show and hide the splash screen manually.
+
+Note that this plugin is not about native loading screens, like e.g. ["Launch Screens" of iOS](https://developer.apple.com/design/human-interface-guidelines/ios/icons-and-images/launch-screen/). This plugin's splash screens are shown _after_ the native app has already started.
```
Comment (timbru31) [inline README.md]: Isn't this only partially true? Android per se does not have any splash screens (at least you have to create a small Activity for this - in contrast Apple forces you to include these launch images)

File: README.md [hunk 3]
```diff
@@ -27,7 +27,26 @@ description: Control the splash screen for your app.
...
+
+- [Installation](#installation)
+- [Supported Platforms](#supported-platforms)
```
Comment (timbru31) [inline README.md]: Shouldn't we be consistent wit upper/lowercase in the heading?

File: README.md [hunk 4]
```diff
@@ -41,10 +60,94 @@ This plugin is required to work with splash screens. This plugin displays and hi
...
+    <splash src="res/screen/ios/Default@2x~iphone.png" width="640" height="960"/>
+    <splash src="res/screen/ios/Default-Portrait~ipad.png" width="768" height="1024"/>
+    <splash src="res/screen/ios/Default-Portrait@2x~ipad.png" width="1536" height="2048"/>
+    <splash src="res/screen/ios/Default-Landscape~ipad.png" width="1024" height="768"/>
+    <splash src="res/screen/ios/Default-Landscape@2x~ipad.png" width="2048" height="1536"/>
+    <splash src="res/screen/ios/Default-568h@2x~iphone.png" width="640" height="1136"/>
+    <splash src="res/screen/ios/Default-667h.png" width="750" height="1334"/>
+    <splash src="res/screen/ios/Default-736h.png" width="1242" height="2208"/>
+    <splash src="res/screen/ios/Default-Landscape-736h.png" width="2208" height="1242"/>
+    <!-- Storyboard method (supports all devices):
```
Comment (timbru31) [inline README.md]: Personally I'd move the Storyboard method before the legacy method in order to encourage users to use this method instead.

File: README.md [hunk 5]
```diff
@@ -428,15 +456,15 @@ In your `config.xml`, you can add the following preferences:
...
 
-The plugin reloads splash drawable whenever orientation changes, so you can specify different drawables for portrait and landscape orientations.
+The splash screen plugin reloads the splash screen whenever the orientation changes so that you can specify different splash screen images for portrait and landscape orientations.
 
-"SplashShowOnlyFirstTime" preference is also optional and defaults to `true`. When set to `true` splash screen will only appear on application launch. However, if you plan to use `navigator.app.exitApp()` to close application and force splash screen appear on next launch, you should set this property to `false` (this also applies to closing the App with Back button).
+`SplashShowOnlyFirstTime` preference is  optional and defaults to `true`. When set to `true` the splash screen will only appear on application launch. However, if you plan to use `navigator.app.exitApp()` to close the application and force the splash screen appear on the application's next launch, you should set this property to `false` (this also applies to closing the application with the Back button).
```
Comment (timbru31) [inline README.md]: `back`

File: README.md (hunk 2 above)
Comment (janpio) [reply README.md]: What you write is true, but it doesn't make what I wrote wrong, does it? If you use a splash theme or splash activity on Android, this plugin's splash screen will still only be shown after that is gone. And in general, this plugin only kicks in when the native app has been started.

File: README.md (hunk 5 above)
Comment (janpio) [reply README.md]: instead of "Back button"?

File: README.md (hunk 3 above)
Comment (janpio) [reply README.md]: Could be, feel free to provide suggestions via the "Insert a suggestion" feature in the comment functionality here where stuff should be changed.

File: README.md (hunk 4 above)
Comment (janpio) [reply README.md]: which one is default for new projects currently?

File: README.md (hunk 5 above)
Comment (timbru31) [reply README.md]: yes

File: README.md (hunk 3 above)
Comment (janpio) [reply README.md]: Done for the headlines that are listed in the ToC

File: README.md (hunk 5 above)
Comment (janpio) [reply README.md]: Just checked, Android usually calls them `Back button`: https://developer.android.com/guide/navigation/navigation-custom-back So I think we should use that as well.

File: README.md (hunk 2 above)
Comment (janpio) [reply README.md]: What about just removing this sentence/paragraph again? It might be more confusing than that it helps.

Comment (janpio) [reply README.md]: Removed for now, created #203 for later.

File: README.md (hunk 4 above)
Comment (timbru31) [reply README.md]: I don't know tbh

Comment (janpio) [reply README.md]: Ok, let's just defer this for later. Can you create an issue please to revisit this?
Then also re-review this PR and approve if you don't find anything else. Thanks.

File: README.md (hunk 5 above)
Comment (timbru31) [reply README.md]: Interesting, but fine 👍
